{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"expected [true] but found [false]","children":[{"name":"test","uid":"eea63f330188fc80","parentUid":"067aa11ae245eca950dc984999fefd6d","status":"failed","time":{"start":1739208839553,"stop":1739208839639,"duration":86},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"067aa11ae245eca950dc984999fefd6d"},{"name":"The following asserts failed:\n\tVerify that report does not contain duplicates. expected [true] but found [false]\nStack Trace :[org.testng.Assert.fail(Assert.java:111)\n org.testng.Assert.failNotEquals(Assert.java:1578)\n org.testng.Assert.assertTrue(Assert.java:57)\n org.testng.asserts.Assertion$1.doAssert(Assertion.java:86)\n com.pinger.automation.utils.SoftVerifier.doAssert(SoftVerifier.java:20)\n org.testng.asserts.Assertion.assertTrue(Assertion.java:82)\n com.pinger.automation.core.model.clients.PingerClient.processValidScenario(PingerClient.java:66)\n functional.TC016_VerifyPingerFlowWithDuplicatesTest.test(TC016_VerifyPingerFlowWithDuplicatesTest.java:31)\n java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n java.base/java.lang.reflect.Method.invoke(Method.java:580)\n org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n org.testng.TestRunner.privateRun(TestRunner.java:819)\n org.testng.TestRunner.run(TestRunner.java:619)\n org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n org.testng.SuiteRunner.run(SuiteRunner.java:336)\n org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\n org.testng.TestNG.runSuites(TestNG.java:1134)\n org.testng.TestNG.run(TestNG.java:1101)\n org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)]","children":[{"name":"test","uid":"62c07996d38b2aca","parentUid":"dcfe61ce93aefb1e155abe8dae60d162","status":"failed","time":{"start":1739208879825,"stop":1739208939903,"duration":60078},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"dcfe61ce93aefb1e155abe8dae60d162"},{"name":"The following asserts failed:\n\tAssert endpoint data. expected [EndpointDto(address=9.9.9.9, description=Quad9 DNS, ignore=false)] but found [EndpointDto(address=149.112.112.112, description=Quad9 DNS, ignore=false)]\nStack Trace :[org.testng.Assert.fail(Assert.java:111)\n org.testng.Assert.failNotEquals(Assert.java:1578)\n org.testng.Assert.assertEqualsImpl(Assert.java:150)\n org.testng.Assert.assertEquals(Assert.java:132)\n org.testng.asserts.Assertion$8.doAssert(Assertion.java:156)\n com.pinger.automation.utils.SoftVerifier.doAssert(SoftVerifier.java:20)\n org.testng.asserts.Assertion.assertEquals(Assertion.java:152)\n com.pinger.automation.core.model.clients.PingerClient.validateEndpointEntries(PingerClient.java:88)\n com.pinger.automation.core.model.clients.PingerClient.processValidScenario(PingerClient.java:72)\n functional.TC015_VerifyElevenEndpointsProcessingTest.test(TC015_VerifyElevenEndpointsProcessingTest.java:37)\n java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n java.base/java.lang.reflect.Method.invoke(Method.java:580)\n org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n org.testng.TestRunner.privateRun(TestRunner.java:819)\n org.testng.TestRunner.run(TestRunner.java:619)\n org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n org.testng.SuiteRunner.run(SuiteRunner.java:336)\n org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\n org.testng.TestNG.runSuites(TestNG.java:1134)\n org.testng.TestNG.run(TestNG.java:1101)\n org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)],\n\tAssert endpoint data. expected [EndpointDto(address=1.1.1.1, description=Cloudflare DNS, ignore=false)] but found [EndpointDto(address=1.1.1.1, description=Cloudflare DNS, ignore=true)]\nStack Trace :[org.testng.Assert.fail(Assert.java:111)\n org.testng.Assert.failNotEquals(Assert.java:1578)\n org.testng.Assert.assertEqualsImpl(Assert.java:150)\n org.testng.Assert.assertEquals(Assert.java:132)\n org.testng.asserts.Assertion$8.doAssert(Assertion.java:156)\n com.pinger.automation.utils.SoftVerifier.doAssert(SoftVerifier.java:20)\n org.testng.asserts.Assertion.assertEquals(Assertion.java:152)\n com.pinger.automation.core.model.clients.PingerClient.validateEndpointEntries(PingerClient.java:88)\n com.pinger.automation.core.model.clients.PingerClient.processValidScenario(PingerClient.java:72)\n functional.TC015_VerifyElevenEndpointsProcessingTest.test(TC015_VerifyElevenEndpointsProcessingTest.java:37)\n java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n java.base/java.lang.reflect.Method.invoke(Method.java:580)\n org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n org.testng.TestRunner.privateRun(TestRunner.java:819)\n org.testng.TestRunner.run(TestRunner.java:619)\n org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n org.testng.SuiteRunner.run(SuiteRunner.java:336)\n org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\n org.testng.TestNG.runSuites(TestNG.java:1134)\n org.testng.TestNG.run(TestNG.java:1101)\n org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)],\n\tAssert endpoint data. expected [EndpointDto(address=192.168.0.101, description=My Grandma, ignore=false)] but found [EndpointDto(address=192.168.0.101, description=My Grandma, ignore=true)]\nStack Trace :[org.testng.Assert.fail(Assert.java:111)\n org.testng.Assert.failNotEquals(Assert.java:1578)\n org.testng.Assert.assertEqualsImpl(Assert.java:150)\n org.testng.Assert.assertEquals(Assert.java:132)\n org.testng.asserts.Assertion$8.doAssert(Assertion.java:156)\n com.pinger.automation.utils.SoftVerifier.doAssert(SoftVerifier.java:20)\n org.testng.asserts.Assertion.assertEquals(Assertion.java:152)\n com.pinger.automation.core.model.clients.PingerClient.validateEndpointEntries(PingerClient.java:88)\n com.pinger.automation.core.model.clients.PingerClient.processValidScenario(PingerClient.java:72)\n functional.TC015_VerifyElevenEndpointsProcessingTest.test(TC015_VerifyElevenEndpointsProcessingTest.java:37)\n java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n java.base/java.lang.reflect.Method.invoke(Method.java:580)\n org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n org.testng.TestRunner.privateRun(TestRunner.java:819)\n org.testng.TestRunner.run(TestRunner.java:619)\n org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n org.testng.SuiteRunner.run(SuiteRunner.java:336)\n org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\n org.testng.TestNG.runSuites(TestNG.java:1134)\n org.testng.TestNG.run(TestNG.java:1101)\n org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)]","children":[{"name":"test","uid":"36a198a6b91cc052","parentUid":"7ef64eb3c98121f3ee5410c02913cd05","status":"failed","time":{"start":1739208939925,"stop":1739209360050,"duration":420125},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"7ef64eb3c98121f3ee5410c02913cd05"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}